lib/MusicBrainz/Server/Data/Utils.pm

```diff
@@ -360,6 +361,16 @@ sub trim_comment {
     return trim($t);
 }
 
+sub trim_text {
+    my $t = shift;
+
+    # Not trimming starting spaces to avoid breaking list formatting,
+    # consider trimming again once this uses Markdown
```

Comment:
"once this uses Markdown" is less clear in here than in Form::Annotation. Should this be called `trim_annotation` and say "once annotations use Markdown"?

Reply:
Annotation has a specific syntax for now but there is nothing specific to annotation’s syntax in this function.
It could (or should?) also be used for other text fields such as editor’s bio, edit notes.
Eventually, all of these fields will move to Markdown syntax.

Reply:
The only issue I have is that it's not distinguished from `trim` (which also operates on text). So I wouldn't know what the difference between `trim` and `trim_text` are from the names alone, I'd have to check usages and git history. What about something like `trim_wikitext`?

Reply:
Renamed as `trim_multiline_text` in commit https://github.com/metabrainz/musicbrainz-server/pull/1944/commits/2b4597fa01406cc296545c4bbbcd0fd6a1c8b647 since it can be used for editor’s biography, edit note, event setlist, description… which are not necessarily wikitext.

Reply:
Then I'd suggest "consider trimming again once text fields currently using wikitext start using Markdown" or whatnot maybe?

Reply:
Actually I see you changed this further down the line so it's probably fine :)

Reply:
Yes, even in Markdown we may want to preserve leading spaces.